docs/backup_and_restore.rst

```diff
@@ -123,67 +105,216 @@ Restoring
 Prerequisites
 '''''''''''''
 
-The process for restoring a backup is very similar to the process of creating
-one. As before, boot the *Admin Workstation* and ``cd`` to the
-SecureDrop repository. Ensure that you have SecureDrop 0.4 or later
-checked out.
+To perform a restore, boot into the *Admin Workstation* and
+ensure that your ``.tar.gz`` backup archive has been copied to
+``~/Persistent/securedrop/install_files/ansible-base``.
+(If you are using the same *Admin Workstation* as you did when you took the backup,
+the archive will already be in place).
 
-The restore command expects to find a ``.tar.gz`` backup archive in
-``install_files/ansible-base`` under the SecureDrop repository root directory.
-If you are using the same *Admin Workstation* to do a restore from a previous
-backup, it should already be there because it was placed there by the backup
-command. Otherwise, you should copy the backup archive that you wish to restore to
-``install_files/ansible-base``.
+If you are restoring data onto an existing instance whose onion URLs have not
+changed (for example, for data recovery purposes), see :ref:`Restoring Data on an
+Existing SecureDrop Installation <restore_data>`.
+
+If you are restoring data onto an instance whose onion URLs *have* changed,
+or you are performing a hardware migration, see :ref:`Migrating <migrating>`. 
+
+If your instance has switched from v2 to v3 onion services since your
+most recent backup, see the section 
+:ref:`on mismatched Tor configurations <restore-with-mismatched-tor-config>`, 
+or  `contact Support`_.
 
 .. note:: The backup strategy used for SecureDrop versions prior to 0.3.7
           created encrypted archives with the extension ``.zip.gpg``.
           You can safely remove those files once you've created the ``.tar.gz``
```

Comment:
^ SecureDrop 0.3.7 is a long time ago, I'd nominate this block for removal while we're overhauling this chapter.

Reply:
Removed in 151175d